src/pages/offer-screen/offer-screen.tsx

```diff
@@ -28,7 +35,19 @@ function OfferScreen({offers, feedbacks}: OfferScreenProps): JSX.Element {
     return <Navigate to="/404" replace />;
   }
 
-  const { bedrooms, goods, host, images, description, isPremium, maxAdults, price, rating, title, type} = offer;
+  const nearestRooms = offers.filter((room) => room.id !== +id);
+
+  function getNearestPoints(arr: Offer[]) {
+    const result: Set<PointType> = new Set();
+    for (const point of arr) {
+      result.add(point.location);
+    }
+    return result;
+  }
+
+  const nearestPoints = getNearestPoints(offers);
```

Comment:
в целом учитывая что getNearestPoints вызывается только один раз, тут проще сделать через map 
```suggestion
  const nearestPoints = offers.map(offer => offer.location);
```